src/Paprika/Store/FanOutList.cs

```diff
@@ -74,4 +74,19 @@ public void Report(IReporter reporter, IPageResolver resolver, int level)
             }
         }
     }
+    public void Destroy(IBatchContext batch, in NibblePath prefix)
+    {
+        // Destroy the Id entry about it
+        Set(prefix, ReadOnlySpan<byte>.Empty, batch);
+
+        // Destroy the account entry
+        // SetAtRoot<FanOutPage>(batch, account, ReadOnlySpan<byte>.Empty, ref Data.StateRoot);
+
+        // Remove the cached
+        batch.IdCache.Remove(prefix.UnsafeAsKeccak);
```

Comment:
I think, the removal from `batch.IdCache` should be only in the `RootPage`, once the mapping is removed, we're ok and requires no more removals.